packages/editor/src/components/Typography.js

```diff
@@ -0,0 +1,8 @@
+import React, { forwardRef } from 'react'
+
+export const Heading = forwardRef(
+  ({ as, level, marginBottom, marginTop, ...props }, ref) => {
```

Comment:
Might be simpler to use HTML elements instead of this component – it doesn't look like it's adding any styles or anything is it?